Client/reasn-client/apps/web/app/register/user/page.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import {
+  ButtonBase,
+  FloatingInput,
+} from "@reasn/ui/src/components/shared/form";
+import React, { useRef, useState } from "react";
+
+const RegisterUser = () => {
+  const [currentStep, setCurrentStep] = useState(1);
+  const formRef = useRef<HTMLFormElement>(null);
+
+  const handleFormSubmit = () => {
+    console.log("form submitted");
+    formRef.current?.submit();
+  };
+
+  return (
+    <>
+      <div className="flex w-1/2 items-center justify-center">
+        <form className="flex w-full flex-col gap-8" ref={formRef}>
+          {currentStep === 1 && (
+            <>
+              <FloatingInput type="text" label="imię" />
+              <FloatingInput type="text" label="nazwisko" />
+              <FloatingInput type="text" label="nazwa użytkownika" />
+              <FloatingInput type="email" label="email" />
+              <FloatingInput type="tel" label="numer telefonu" />
+              <FloatingInput type="password" label="hasło" />
```

Comment:
Add field for confirming password - it will prevent the user from making mistake when entering the password